Add Export & Import release bundle docs (#107)
jfrog/documentation: jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory/release-lifecycle-management.md

|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
| Command-name           | release-bundle-export                                                                                                                           |
| Abbreviation           | rbe                                                                                                                                             |
| Command arguments      |                                                                                                                                                 |
| release bundle name    | Name of the release bundle to distribute.                                                                                                       |
| release bundle version | Version of the release bundle to distribute.                                                                                                    

> | target pattern         | Specifies the local file system target download path                                                                                                     

Reviewer: I suggest to put the whole explanation as in the help because users might relay on documentation and not the help command

Reviewer: Same comment for `mapping-pattern` and `mapping-target`.

Also, missing `split-count` and `min-split`

| Command options        |                                                                                                                                                 |
| --project              | <p>[Optional]<br><br>Project key associated with the Release Bundle version.</p>                                                                |
| --server-id            | <p>[Optional]<br><br>Platform server ID configured using the config command.</p>                                                                |
| mapping-pattern        | <p>[Optional]<br><br> Specify a list of input regex mapping pairs that define where the queried artifact is located and where it should be placed after it is imported                                                                                                   
| mapping-target         | <p>[Optional]<br><br> Specify a list of output regex mapping pairs that define where the queried artifact is located and where it should be placed after it is imported

#### Example
Export release bundle named "myApp" and version 1.0.0

	jf rbe myApp 1.0.0

#### Example
Download to a specific location

	jf rbe myApp 1.0.0 /user/mybundle/

> ## Importing Release Bundle Archive

Reviewer: ```suggestion
## Importing a Release Bundle archive
```

Import a Release Bundle archive from a release bundle exported zip file.

> Please note this functionality only works on Edge nodes within an Air Gap environment.

Reviewer: Isn't the only constraint is being an Edge node (defined by license)?
```suggestion
Please note this functionality only works on Edge nodes within an air-gapped environment.
```

|                        |                                                                                  |
|------------------------|----------------------------------------------------------------------------------|
| Command-name           | release-bundle-import                                                            |
| Abbreviation           | rbi                                                                              |
| Command arguments      |                                                                                  |

> | path to archive        | path to the release bundle archive on the filesystem                             | 

Reviewer: ```suggestion
| path to archive        | Path to the release bundle archive on the filesystem                             | 
```

| Command options        |                                                                                  |
| --project              | <p>[Optional]<br><br>Project key associated with the Release Bundle version.</p> |
| --server-id            | <p>[Optional]<br><br>Platform server ID configured using the config command.</p> |

#### Example

> Export release bundle named "myExportedApp" and version 1.0.0

Reviewer: ```suggestion
Import a Release Bundle named "myExportedApp" and version 1.0.0
```
